man/udev.xml

```diff
@@ -236,6 +236,31 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>CONST{<replaceable>key</replaceable>}</varname></term>
+          <listitem>
+            <para>Match against a system-wide constant. Supported keys are:</para>
```

Comment:
What happens when an unknown key is used must be specified here. I can think of two choices:
1. matches for unknown keys are ignored, i.e. they always return true
2. they always return false and never match.

Option 2. seems more appropriate.